<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Swimming Lake Management</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="titleLabel">
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Swimming Lake Management System</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="infoLayout">
     <item>
      <widget class="QGroupBox" name="lakeInfoGroup">
       <property name="title">
        <string>Lake Information</string>
       </property>
       <layout class="QVBoxLayout" name="lakeInfoLayout">
        <item>
         <widget class="QLabel" name="temperatureLabel">
          <property name="text">
           <string>Water Temperature: -- Â°C</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="qualityLabel">
          <property name="text">
           <string>Water Quality: --</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="visitorsLabel">
          <property name="text">
           <string>Current Visitors: --</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="actionsGroup">
       <property name="title">
        <string>Actions</string>
       </property>
       <layout class="QVBoxLayout" name="actionsLayout">
        <item>
         <widget class="QPushButton" name="rentLoungerButton">
          <property name="text">
           <string>Rent Lounger</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="parkBikeButton">
          <property name="text">
           <string>Park Bike</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="refreshButton">
          <property name="text">
           <string>Refresh Info</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextEdit" name="logTextEdit">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
